lib/cartopy/crs.py

```diff
@@ -1454,7 +1454,7 @@ def __init__(self, central_longitude=0, globe=None):
         # 40 deg N introduced by incomplete fix to issue #113 (see
         # https://trac.osgeo.org/proj/ticket/113).
         import re
-        match = re.search(r"\d\.\d", PROJ4_RELEASE)
+        match = re.search(r"\d+\.\d+", PROJ4_RELEASE)
         if match is not None:
             proj4_version = float(match.group())
             if 4.8 <= proj4_version < 4.9:
```

Comment:
Good point about the `float`. Actually, there are a couple more cases of this in the tests that I need to fix as well.